api/src/models/Job.test.ts

```diff
@@ -0,0 +1,27 @@
+import Job from "./Job";
+import Config from "../models/Config";
+import QueueManager from "../services/QueueManager";
+
+jest.mock("../models/Config");
+jest.mock("../services/QueueManager");
+
+describe("Job", () => {
+    let job: Job;
+    let config;
+    let queueManager;
+    beforeEach(() => {
+        config = { };
+        queueManager = { 
+            performIndexOperation: jest.fn(),
```

Comment:
I wouldn't bother mocking these methods until we actually have a test that is going to call them.

Reply:
fixed